scripts/qa_crowbarsetup.sh

```diff
@@ -3910,6 +3910,10 @@ function onadmin_runupdate()
 
     pre_hook $FUNCNAME
 
+    # We need to set the correct MTU here since we haven't done any
+    # proper network configuration yet.
+    ip link set mtu $host_mtu dev eth0
```

Comment:
maybe add a [[ $host_mtu ]] &&
because when deploying physical clouds, we dont have the mkcloud part.